app/assets/javascripts/Components/graders_manager.jsx

```diff
@@ -156,11 +160,51 @@ class GradersManager extends React.Component {
   toggleAssignGradersToCriteria = () => {
     const assign = !this.state.assign_graders_to_criteria;
     $.post({
-      url: Routes.set_assign_criteria_assignment_graders_path(this.props.assignment_id),
-      data: {value: assign},
+      url: Routes.set_boolean_graders_options_assignment_path(this.props.assignment_id),
+      data: {
+        value: assign,
+        attribute: 'assign_graders_to_criteria'
```

Comment:
Good idea unifying the routes here. One comment I have is you should be able to mimic the typical attribute update params, e.g. `data: {properties: {assign_graders_to_criteria: assign}}`, and then in the `GradersController` first do a `params.permit` and then simply pass the whitelisted params to `Assignment.update` directly.